.github/workflows/cross_platform_ci.yml

```diff
@@ -0,0 +1,101 @@
+name: Build Moveit Core on all platforms
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+      - "[kmn]*-devel"
+
+jobs:
+  run_moveit_core_compilation:
+    runs-on: ${{ matrix.os }}
```

Comment:
```suggestion
    runs-on: ${{ matrix.os }}
    name: ${{ matrix.os }}
```

Reply:
Fixed in https://github.com/ros-planning/moveit/pull/2604/commits/839b4c0c569eb760efe4b5b2a8ffa856fff2c1c5